handling issue 329

naraharip2017 opened the pull request:
<!-- Please describe your pull request here. -->

Handling following issue
- https://github.com/jenkinsci/ec2-fleet-plugin/issues/329

### Testing done

- Verified via UI, `slave` terminology is updated

<!-- Comment:
Provide a clear description of how this change was tested.
At minimum this should include proof that a computer has executed the changed lines.
Ideally this should include an automated test or an explanation as to why this change has no tests.
Note that automated test coverage is less than complete, so a successful PR build does not necessarily imply that a computer has executed the changed lines.
If automated test coverage does not exist for the lines you are changing, you must describe the scenario(s) in which you manually tested the change.
For frontend changes, include screenshots of the relevant page(s) before and after the change.
For refactoring and code cleanup changes, exercise the code before and after the change and verify the behavior remains the same.
-->

```[tasklist]
### Submitter checklist
- [ X ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ X ] Ensure that the pull request title represents the desired changelog entry
- [ X ] Please describe what you did
- [ X ] Link to relevant issues in GitHub or Jira
- [ X ] Link to relevant pull requests, esp. upstream and downstream changes
- [ X ] Ensure you have provided tests - that demonstrates feature works or fixes the issue
```

<!--
Put an `x` into the [ ] to show you have filled the information.
The template comes from https://github.com/jenkinsci/.github/blob/master/.github/pull_request_template.md 
You can override it by creating .github/pull_request_template.md in your own repository 
-->

pdk27 commented on src/main/java/com/amazon/jenkins/ec2fleet/EC2FleetCloud.java:
```diff
@@ -861,7 +861,7 @@ private void addNewSlave(final AmazonEC2 ec2, final Instance instance, FleetStat
                 computerConnector.launch(address, TaskListener.NULL));
         final Node.Mode nodeMode = restrictUsage ? Node.Mode.EXCLUSIVE : Node.Mode.NORMAL;
-        final EC2FleetNode node = new EC2FleetNode(instanceId, "Fleet slave for " + instanceId,
+        final EC2FleetNode node = new EC2FleetNode(instanceId, "Fleet node for " + instanceId,
```
I think using agent instead of node is better as we refer to both node and computer as a slave / agent / worker.
Ref: https://www.jenkins.io/doc/book/managing/nodes/#creating-agents

naraharip2017 replied:
Makes sense, updated

pdk27 commented:
Let's take this a step forward and replace all occurrences of `slave` in the code too - https://github.com/search?q=repo%3Ajenkinsci%2Fec2-fleet-plugin+slave&type=code&p=1.

naraharip2017 commented:
> Let's take this one step further and replace all occurrences of `slave` in the code too - https://github.com/search?q=repo%3Ajenkinsci%2Fec2-fleet-plugin+slave&type=code&p=1.

Updated in revision. Removed all references that are not either a link to a webpage using `slave` terminology or referencing slave objects from the jenkins libraries

pdk27 reviewed (approved):
LGTM 🚀 ⭐
